src/models/grantPeriod/grantPeriodForm.ts

```diff
@@ -1,5 +1,11 @@
 import { FORM_ACTIONS } from "../constants";
 import { IDONOR } from "../donor";
+import { FetchResult } from "@apollo/client";
+
+export enum DonorType {
+	project = "PROJECT'S DONOR",
```

Comment:
These are not part of multi-lingual?

Reply:
Wherever I an displaying the value of enum I have changed that to formatted message